Cluster Programs

```diff
@@ -1,3 +1,73 @@
+August 6/2018
+Compilation about Documentation
+
+1) Documentation about the missing values in nclass was unavailable. 
+This variable comes in the original datasets from Harvard Dataverse, it is NOT created in the SAS code. 
+Possiblenclass has 911 NAs and Matching2 (before its merge with all the if conditions) has 175 NAs. Once its merge, Matching 2 (running all the if conditions) ends up with 94 NAs.
+Now, here I find something strange, the first 16 matches are different, but the rest 78 is the same patent (same cited and patent) that it's been matched to different controls.
+In the next step of the SAS code, it applys nodupkey, dropping all the NA's. I end up with 17 NA's, after the first iteration.
+The code never deals with the NAs, not even in the third program.
+In my opinion, we could drop the NAs in the code, given that we are mainly grouping by nclass and NAs could mean that we are grouping different things 
+(e.g. paring nclass=4 with nclass=400) giving a wrong matching. This is what Thompson (2005) critic about Jaffee's method when he matched 
+the primary class with out considering the subclasses.
+By looking at the SAS code.
+
+3) In the original dataset from Harvard Dataverse you can find the variables Invnum and Invnum_N. 
+Invnum: Initial inventor number: Patent + Invseq. 
+Invnum_N: Disambiguated inventor number. This function assign a number to each of the individual inventors, this way an inventor with more than one patent will have the same number.  
+In the program, we are using Invnum_N, which later it’s rename to Invnum.
+
+4) Next is presenting the differences between both method of matching, Thompson and Jaffe e.t, in order to determined the one we are using.  
+In Thompson 2005 article the different methods are made clear. 
```

Comment:
We should call this TFK (2005) to be clear, since Thomson is not the only author